service/test/POSIXSignalTest.cpp

```diff
@@ -61,6 +69,40 @@ void POSIXSignalTest::TearDown(void)
     sigprocmask(SIG_SETMASK, &m_backup_sigset, nullptr);
 }
 
+bool POSIXSignalTest::has_elevated_permissions(void) const
+{
+    uint64_t cap = 0;
+    uint64_t cap_sys_admin = 0x00200000;  // TODO: magic number, is there a #define for it?
```

Comment:
We probably just want to check for CAP_KILL which is 0x20, but perhaps renaming the function as well "has_cap_kill()"